test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php

```diff
@@ -465,6 +465,33 @@ public function testStubForConstantThatExists(): void
         self::assertSame('Core', $stubData->getExtensionName());
     }
 
+    public function testStubForConstantThatIsDeprecated(): void
+    {
+        $stubData = $this->sourceStubber->generateConstantStub('FILTER_SANITIZE_STRING');
+
+        self::assertInstanceOf(StubData::class, $stubData);
+
+        self::assertStringMatchesFormat(
+            "%Adefine('FILTER_SANITIZE_STRING',%w%d);",
```

Comment:
I'm OK with the patch per se, but this test will start failing when we switch PHP versions, which makes it a bit fragile.

I wonder if we could define a stub that we read just for the purposes of this test? :thinking: 

/cc @kukulich WDYT?

Reply:
good idea. added a fake stub instead.